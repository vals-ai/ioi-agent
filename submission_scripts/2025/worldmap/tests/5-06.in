version https://git-lfs.github.com/spec/v1
oid sha256:4e9a93d1de5a5f89db5b0ae2cd5ad1f087baa328b38bc0a4b00954aeacf66bbb
size 4832
