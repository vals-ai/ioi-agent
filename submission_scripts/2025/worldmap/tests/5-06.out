version https://git-lfs.github.com/spec/v1
oid sha256:03949f80b35b9a456a625cb46d3eb8e218e6d992a38762a51e74cd370809e320
size 22738
